app/Http/Controllers/Settings/UserRoleAssignController.php

```
@@ -0,0 +1,62 @@
+use App\Models\User;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Inertia\Inertia;
+use Inertia\Response;
+
+class UserRoleAssignController
+{
+    public function index(): Response
+    {
+        return Inertia::render('settings/roles', [
+            'users' => User::all()->map(fn($user) => [
```

PHPStan (2.0.3)

Parameter #1 $callback of method Illuminate\Database\Eloquent\Collection<int,App\Models\User>::map() contains unresolvable type.


(argument.unresolvableType)